config/rule.go

```diff
@@ -1,6 +1,11 @@
 package config
 
-import "fmt"
+import (
+	"fmt",
+	"go.uber.org/zap",
+	"go.uber.org/zap/zapcore",
```

Comment:
don't need to introduce external packages.

Reply:
I'll remove these changes